plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestGlueConverter.java

```diff
@@ -346,7 +346,7 @@ void testConvertTableBadHiveType()
     {
         software.amazon.awssdk.services.glue.model.Table table = glueTable.toBuilder()
                 .storageDescriptor(
-                        software.amazon.awssdk.services.glue.model.StorageDescriptor.builder()
```

Comment:
This change should go to "Assert breakages around malformed hive metadata" commit.